lang/src/org/partiql/lang/eval/physical/PhysicalBexprToThunkConverter.kt

```diff
@@ -311,12 +313,26 @@ internal class PhysicalBexprToThunkConverter(
             exprConverter.convert(it).toValueExpr(it.metas.sourceLocationMeta)
         }
 
+        // window function map
+        val builtinWindowFunctions = createBuiltinWindowFunctions()
+        val builtinWindowFunctionsMap = builtinWindowFunctions.associateBy {
+            it.signature.name
+        }
```

Comment:
I'm curious if there's a better way to create compiled window functions. Seemingly, for several nested window operators, we'll be creating all builtin functions even when we don't use them. I'm also curious about the modeling of CompiledWindowFunctions vs WindowFunctions.